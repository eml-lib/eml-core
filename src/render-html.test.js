import renderHtml from './render-html';

it('Self-closing elements', () => {
	expect(
		renderHtml(
			{ tagName: 'div', children: [
				{ tagName: 'br' },
				{ tagName: 'input' },
				{ tagName: 'img', attrs: { src: '' } }
			] }
		)
	).toMatchSnapshot();
});

it('Deep', () => {
	expect(
		renderHtml(
			{ tagName: 'div', children: [
				{ tagName: 'div', children: [
					{ tagName: 'div', children: [ 123 ] },
					{ tagName: 'div', children: [
						'Многострочный\nтекст,\nкоторый\nдолжен\nтабулироваться'
					] }
				] }
			] }
		)
	).toMatchSnapshot()
});