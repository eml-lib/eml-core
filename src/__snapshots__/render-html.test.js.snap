// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deep 1`] = `
"<div>
	<div>
		<div>
			123
		</div>
		<div>
			Многострочный
			текст,
			который
			должен
			табулироваться
		</div>
	</div>
</div>
"
`;

exports[`Self-closing elements 1`] = `
"<div>
	<br />
	<input />
	<img src=\\"\\" />
</div>
"
`;
